policyengine_us/variables/gov/states/sc/tanf/sc_tanf_fpg.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class sc_tanf_fpg(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "South Carolina TANF federal poverty guideline"
+    unit = USD
+    defined_for = StateCode.SC
+    definition_period = MONTH
```

Comment:
If the income eligible and the actual TANF files are yearly, would it not enough to make this yearly and just not divide lines 27 and 28 by the Months

Reply:
The results would be different, I think when the definition period is MONTH, it calculates each month and adds them all up. In a case where we don't need October as the breakdown month, it should not matter. But since we are defining January through October as previous year and the rest as current year in the formula, I think we have to use month. 
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/bddf89f2-33bc-4eb8-b9d4-57934abc83be)

Reply:
I see, yes I definitely agree.